crates/sui-benchmark/src/drivers/mod.rs

```diff
@@ -119,6 +131,38 @@ impl BenchmarkStats {
         table.add_row(row);
         table
     }
+
+    pub fn to_cpu_table(&self) -> Table {
+        let mut table = Table::new();
+        table
```

Comment:
nit: I think we don't need this many quantiles for CPU or latency. For CPU just average would be good enough. For latency, min, p50 and p99 should be good enough.

Per-core CPU info may be a bit too much as well. Is it easy to calculate avg CPU across all cores over the period, and move CPU and latency to a single table?